cmd/kubectl-datadog/flare/flare.go

```diff
@@ -132,7 +132,7 @@ func (o *options) complete(cmd *cobra.Command, args []string) error {
 	}
 
 	if ddSite == "" {
-		ddSite, err = common.AskForInput("Please enter your Datatod site [us/eu] (default us): ")
+		ddSite, err = common.AskForInput("Please enter your Datatog site [us/eu] (default us): ")
```

Comment:
Fixing a typo in the flare sending process. Happy to make this change in a dedicated PR.